server/controllers/user.controller.js

```diff
@@ -24,11 +24,19 @@ export function createUser(req, res, next) {
           if (loginErr) {
             return next(loginErr);
           }
-          res.json({
-            email: req.user.email,
-            username: req.user.username,
-            preferences: req.user.preferences,
-            id: req.user._id
+          mail.send('email-verification', {
+            body: {
+              link: `http://${req.headers.host}/api/verify?t=${auth.createVerificationToken(req.body.email)}`
```

Comment:
No. Remember that this app is all front-end communicating with an API, therefore the redirection should be on the front end. Once the status of the verification is received, the user will be redirected to a verification status/success/whatever makes sense page.